README.md

```
@@ -1,267 +1,216 @@
 # Gradata
 
-## AI that learns your judgment, not just your preferences.
+### Your AI keeps making the same mistakes. Gradata makes it stop.
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)